paywall/src/messageTypes.ts

```diff
@@ -108,7 +111,7 @@ export type Message =
     }
   | {
       type: PostMessages.UPDATE_NETWORK
-      payload: number
+      payload: unlockNetworks
```

Comment:
This is a small change. Previously we were coercing numbers to be this type for use in this payload.